Docs/source/Software/drivers.rst

```diff
@@ -77,10 +77,20 @@ SD Card
 =======
 
 Purpose
+The SD Card is used to collect data in the event that it could not be sent to the data acquisition system over ethernet. This means that
+data is logged periodically and can be parsed later.
 
 Usage
+``SDCard_Init()`` mounts the SD card and should be called before any other function. ``SDCard_GetStatistics()`` should be used when debugging
+and not in the main loop (unless to check if there is not enough data left). ``SDCard_PutInQueue()`` should be used by the thread placing
+data in the queue while ``SDCard_Sort_Write_Data()`` will take the data from the queue or return an error if there is no data. The error
+does not pertain to an empty queue. ``SDCard_t`` has a time component that should be sampled from the GPS drivers. In order to prevent data
```

Comment:
the line about ```SDCard_t``` probably should go somewhere else, it doesnt really fit the flow of this paragraph. also, it might help to have something about the fields inside the sdcard struct somewhere